php/php.code.analysis/src/org/netbeans/modules/php/analysis/ui/analyzer/PHPStanCustomizerPanel.java

```diff
@@ -221,6 +216,10 @@ private void initComponents() {
         phpStanLevelComboBox = new javax.swing.JComboBox<>();
         phpStanMemoryLimitLabel = new javax.swing.JLabel();
         phpStanMemoryLimitTextField = new javax.swing.JTextField();
+        phpStanLabel = new javax.swing.JLabel();
```

Comment:
Could we avoid using FQN here in this panel, please? It makes the code more readable, I believe. Thank you.